markdown/dev/guides/patterns/stacks/en.md

```diff
@@ -46,12 +46,21 @@ to patterns designers who want to use it.
 
 Essentially, stacks behave as layers. Parts that are on the same _stack_ will be stacked on top of each other in the layout.
```

Comment:
I think this sentence actually confuses the matter. possibly an improvement: "Essentially a stack is a collection of parts. Parts in a stack act as layers, and will be stacked directly on top of each other."

Reply:
Added.